docs/index.md

```diff
@@ -1,21 +1,34 @@
-# What is Copilot Developer Camp?
+**Copilot Developer Camp** is a self-paced learning resource, designed to help you build agents for Microsoft 365 Copilot. It provides practical guidance and real-world examples to support your development journey.
 
-Copilot Developer Camp is a workshop for makers and professional developers who want to learn how to build agents for Microsoft 365 Copilot.
+# Building Agents for Microsoft 365
 
-  <div class="video">
-      <iframe src="//www.youtube.com/embed/uLYdP4ST7k0" frameborder="0" allowfullscreen></iframe>
-      <div>Overview of Copilot Dev Camp</div>
-    </div>
+Agents for Microsoft 365 are conversational AI-powered assistants designed to help users automate tasks, answer questions, and streamline workflows across Microsoft 365 apps and services. Agents can be used in various scenarios, such as customer support, IT helpdesk, and HR support.
```

Comment:
I would say "such as custom support, IT helpdesk, HR support, and more." just to avoid limiting possibilities.